requirements.txt

```diff
@@ -248,3 +248,5 @@ pyinotify==0.9.6 ; sys_platform == 'linux' or sys_platform == 'linux2' \
     --hash=sha256:9c998a5d7606ca835065cdabc013ae6c66eb9ea76a00a1e3bc6e0cfe2b4f71f4
 django-jinja==2.1.2 \
     --hash=sha256:fc2b6a65201b27711e800eb2e1262ff42a32f027bd2267e34121f1bd2bd6b933
+socorrolib==0.2.2 \
```

Comment:
I had the same thought when I started. Python conventions prevent me from doing that, though. Module and package names are lower case letters with underscores allowed sometimes. Putting in hyphens breaks setup().

I suspect -external will dissolve into individual apps, with just a few common things ending up socorrolib. -cron and -database will likely live together in some kind of admin app.